examples/assets_smoke_test/assets_smoke_test_tests/smoke_test_python_and_dbt_assets.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import snowflake.connector
+from assets_smoke_test import python_and_dbt_assets
+from assets_smoke_test.python_and_dbt_assets import (
+    DBT_PROFILES_DIR,
+    DBT_PROJECT_DIR,
+    raw_country_populations,
+)
+from dagster_dbt import dbt_cli_resource
+from dagster_snowflake import build_snowflake_io_manager
+from dagster_snowflake_pandas import SnowflakePandasTypeHandler
+
+from dagster import load_assets_from_modules, materialize
+
+
+def test_smoke_all():
+    snowflake_config = {
+        "user": os.environ["SNOWFLAKE_USER"],
+        "password": os.environ["SNOWFLAKE_PASSWORD"],
+        "account": os.environ["SNOWFLAKE_ACCOUNT"],
+    }
+
+    database_name = os.environ["SNOWFLAKE_DATABASE"]
```

Comment:
I see that database_name gets used later on, but I think it might be easier to read this test if the database is added to snowflake_config here, then pulled out of the dictionary on line 39.